### PR TITLE
SUMO-218223 Add the details about AWS regions info

### PR DESCRIPTION
**Pull request by yuting-liu**
## Purpose of this pull request

This pull request (PR) adds a table that lists the deployment and the aws region where Sumo's Privatelink service is set up. By this table, the user can reference it to know when the vpc peering is needed for them.

Issue number: 
SUMO-218223

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [x] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/integrations/amazon-aws/aws-privatelink.md [hunk 1]
```diff
@@ -18,7 +18,75 @@ To send data to Sumo Logic through AWS PrivateLink, you'll need to configure an
 
 With the NLB-created and ALB-registered as a target, requests over AWS PrivateLink to the NLB are forwarded to the ALB.
 
-Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend on your Sumo Logic deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security). If you're using the VPC in a different region, you need to set up VPC peering. Either way, you need to create an endpoint.
+Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend on your Sumo Logic deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security). If you're using the VPC in a different region where Sumo's Privatelink endpoint service is set up, you need to set up VPC peering. Either way, you need to create an endpoint.
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-privatelink.md:21]:
```suggestion
Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend on your Sumo Logic deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security). If you're using the VPC in a different region where the Sumo Logic PrivateLink endpoint service is set up, you need to set up VPC peering. Either way, you need to create an endpoint.
```

File: docs/integrations/amazon-aws/aws-privatelink.md [hunk 2]
```diff
@@ -18,7 +18,75 @@ To send data to Sumo Logic through AWS PrivateLink, you'll need to configure an
...
+    <td>https://collectors.us2.sumologic.com<br/>
+https://endpoint1.collection.us2.sumologic.com<br/>
+https://endpoint2.collection.us2.sumologic.com<br/>
+https://endpoint3.collection.us2.sumologic.com<br/>
+https://endpoint4.collection.us2.sumologic.com<br/>
+https://endpoint5.collection.us2.sumologic.com<br/>
+https://endpoint6.collection.us2.sumologic.com<br/>
+https://endpoint7.collection.us2.sumologic.com<br/>
+https://endpoint8.collection.us2.sumologic.com<br/>
+https://endpoint9.collection.us2.sumologic.com/</td>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-privatelink.md:86]:
```suggestion
https://endpoint9.collection.us2.sumologic.com</td>
```

File: docs/integrations/amazon-aws/aws-privatelink.md [hunk 3]
```diff
@@ -18,7 +18,75 @@ To send data to Sumo Logic through AWS PrivateLink, you'll need to configure an
...
 With the NLB-created and ALB-registered as a target, requests over AWS PrivateLink to the NLB are forwarded to the ALB.
 
-Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend on your Sumo Logic deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security). If you're using the VPC in a different region, you need to set up VPC peering. Either way, you need to create an endpoint.
+Sumo Logic exposes AWS PrivateLink endpoints to different [regions that depend on your Sumo Logic deployment](/docs/api/getting-started/#sumo-logic-endpoints-by-deployment-and-firewall-security). If you're using the VPC in a different region where Sumo's Privatelink endpoint service is set up, you need to set up VPC peering. Either way, you need to create an endpoint.
+
+<table><small>
+  <tr>
+    <td>Deployment</td>
+    <td>Collection Endpoint</td>
+    <td>AWS Region of Sumo PrivateLink Endpoint Service</td>
```
Comment (kimsauce) [inline docs/integrations/amazon-aws/aws-privatelink.md:27]:
```suggestion
    <td><strong>Deployment</strong></td>
    <td><strong>Collection Endpoint</strong></td>
    <td><strong>AWS Region of Sumo PrivateLink Endpoint Service</strong></td>
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
formatting

**Review by kimsauce — APPROVED** [review]

**Comment by kimsauce** [comment]
Thank you, @yuting-liu ! May I push this to production?

**Comment by yuting-liu** [comment]
I just merged it. @kimsauce Thank you for approval.

**Comment by yuting-liu** [comment]
@kimsauce Yea. sure. This should be good to be pushed to production

**Pull request merged**